src/services/api.js

```diff
@@ -0,0 +1,19 @@
+import HubsApi from '../hubsApi';
+
+export default class API {
+  /** @type HubsAPI */
+  static hubsApi = null;
+
+  static async setup() {
+    API.hubsApi = new HubsApi(10000);
+    await API.hubsApi.connect('ws://192.168.0.15:8844/');
```

Comment:
change the `ip` for the one shown in the console where you are running the server (in the future we will need to create a script to automatically update this value)